--- source.osim
+++ model.osim
@@ -9125,7 +9125,7 @@
 									<!--The wrapping method used to solve the path around the wrap object.-->
 									<method>hybrid</method>
 									<!--The range of indices to use to compute the path over the wrap object.-->
-									<range>-1 -1</range>
+									<range>1 2</range>
 								</PathWrap>
 								<PathWrap name="pathwrap_0">
 									<!--A WrapObject that this PathWrap interacts with.-->
@@ -9133,7 +9133,7 @@
 									<!--The wrapping method used to solve the path around the wrap object.-->
 									<method>hybrid</method>
 									<!--The range of indices to use to compute the path over the wrap object.-->
-									<range>-1 -1</range>
+									<range>3 4</range>
 								</PathWrap>
 							</objects>
 							<groups />
